apps/shared/lib/schemas.ex

```diff
@@ -70,6 +70,23 @@ defmodule Transport.Shared.Schemas do
     )
   end
 
+  def documentation_url(schema_name), do: documentation_url(schema_name, nil)
+
+  def documentation_url(schema_name, nil = _schema_version) do
+    _ = Map.fetch!(Wrapper.transport_schemas(), schema_name)
+    "https://schema.data.gouv.fr/#{schema_name}/"
+  end
+
+  def documentation_url(schema_name, schema_version) when not is_nil(schema_version) do
```

Comment:
```suggestion
  def documentation_url(schema_name, schema_version) do
```
Le check `when not is_nil` n'est pas nécessaire vu les clauses au dessus